charts/cloudnative-pg/values.yaml

```
@@ -139,6 +139,16 @@ affinity: {}
 monitoring:
   # -- Specifies whether the monitoring should be enabled. Requires Prometheus Operator CRDs.
   podMonitorEnabled: false
+  grafanaDashboard:
+    enabled: false
```

In the rest of the values where we allow specifying whether we have to create some resource from the helm chart we use `create`, `enabled` is more for stuff the operator will then handle:
```suggestion
    create: false
```